R/shuffle.R

```diff
@@ -90,10 +90,11 @@ bake.step_shuffle <- function(object, new_data, ...) {
 
   if (length(object$columns) > 0) {
     for (i in seq_along(object$columns)) {
-      new_data[, object$columns[i]] <-
-        sample(getElement(new_data, object$columns[i]))
+      new_data[[object$columns[i]]] <-
+        sample(new_data[[object$columns[i]]])
     }
```

Comment:
Some of these changes doesn't give super nice readability right now as I find `]]]` a little hard to read.

I will address these in a future PR where I will refactor many of the loops for readability (and speed)

```r
# Old
if (length(object$columns) > 0) {
  for (i in seq_along(object$columns)) {
      new_data[[object$columns[i]]] <-
        sample(new_data[[object$columns[i]]])
    }
}

# New
col_names <- object$columns
for (col_name in col_names) {
  new_data[[col_name]] <- sample(new_data[[col_name]])
}
```